src/js/utils/strings.ts

```diff
@@ -1,12 +1,12 @@
-import { isValidNumber, isString } from 'utils/underscore';
+import { isValidNumber, isString } from './underscore';
```

Comment:
did we figure out the symlinking here? where `utils` points to the proper dir (especially across jwplayer and commercial)

Reply:
Looks like this 'baseUrl' compiler option in the tsconfig will help
https://www.typescriptlang.org/docs/handbook/module-resolution.html#base-url